master/buildbot/status/web/builder.py

```diff
@@ -595,13 +595,13 @@ def stopselected(self, req):
 
 # /builders
 class BuildersResource(HtmlResource):
-    pageTitle = "Katana - Builders"
     addSlash = True
 
     def __init__(self, project, numbuilds=15):
         HtmlResource.__init__(self)
         self.project = project
         self.numbuilds = numbuilds
+        self.pageTitle = self.getBranchName() + "Builders"
```

Comment:
if the user has a different branch selected (and not the default one) this approach won't work. @sualitu I'll revert this logic from the pr.